src/WebJobs.Extensions.DurableTask/EndToEndTraceHelper.cs

```diff
@@ -284,6 +284,56 @@ public void ExtensionWarningEvent(string hubName, string functionName, string in
                 LocalSlotName, ExtensionVersion, this.sequenceNumber++);
         }
 
+        public void SuspendingOrchestration(
+            string hubName,
+            string functionName,
+            string instanceId,
+            string reason)
+        {
+            FunctionType functionType = FunctionType.Orchestrator;
+
+            EtwEventSource.Instance.SuspendingOrchestration(
+                hubName,
+                LocalAppName,
+                LocalSlotName,
+                functionName,
+                instanceId,
+                reason,
+                functionType.ToString(),
+                ExtensionVersion,
+                IsReplay: false);
+
+            this.logger.LogWarning(
```

Comment:
This should be an informational trace rather than a warning. Same for `ResumingOrchestration`.

Reply:
Fixed.